rest/rest.go

```diff
@@ -38,6 +38,7 @@ type Endpoint struct {
 
 	AllowedDuringShutdown bool // Whether we should return Unavailable Error (503) if daemon is shutting down.
 	AllowedBeforeInit     bool // Whether we should return Unavailabel Error (503) if the daemon has not been initialized (is not yet part of a cluster).
+	AllowNoPrefix         bool // Whether we should allow the endpoint to be registered without the /1.0 prefix.
```

Comment:
Looks like this commit is redundant now since you undo the change in a later commit. Could you please clean it up?